packages/inter-protocol/test/reserve/test-reserve.js

```diff
@@ -114,14 +115,6 @@ test('check allocations', async t => {
     `added moola to the collateral Reserve`,
   );
 
-  t.deepEqual(
-    await E(reserve.reserveCreatorFacet).getAllocations(),
-    harden({
-      Moola: moola(10_000n),
-    }),
-    'expecting more',
-  );
```

Comment:
why remove these test assertions?

I'd like to think the contract passes the old tests.

Reply:
It was redundant with the next line that checks the allocation using metrics data.

I had removed getAllocations() because it's redundant with the metrics, but there are so many dependencies on it that I'm restoring it for now. I'll restore this test.